OWNERS_ALIASES

```diff
@@ -314,3 +314,9 @@ aliases:
   - jshaughn
   - FilipB
   - jmazzitelli
+  jboss-fuse-approvers:
+  - smongiar
+  - mcarlett
+  - fmariani
+  - ppinkas
+  - jvrubel
```

Comment:
@smongiar Can you please make sure that above team members also add themselves as a member of Openshift Organisation. They can follow [this](https://cspi-qe.gitbook.io/ocp-ci-onboarding/onboarding/onboarding_guide/prerequisites_guide#github-user-readiness) guide link fo the steps.
This will remove the label:do-not-merge/invalid-owners-file

Reply:
Sure, I will ask other team members